dev-requirements.txt

```diff
@@ -1,18 +1,14 @@
 pytest
-doctr
+pytest-cov
 ipython
```

Comment:
`ipython` is used only in the docs, isn't it?

Reply:
We also use it in the [cli tests](https://github.com/pcdshub/happi/blob/a985ac79407593cf022ddbc1b5eb4aff305dfa14/happi/tests/test_cli.py#L12).